backend/services/security.py

```
@@ -0,0 +1,316 @@
+import hashlib
+import secrets
+import re
+from typing import Optional, Dict, Any
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import

(F401)